wsd/FileServer.cpp

```diff
@@ -562,7 +562,8 @@ void FileServerRequestHandler::handleRequest(const HTTPRequest& request,
             endPoint == "uno-localizations.json" ||
             endPoint == "uno-localizations-override.json")
         {
-            preprocessFile(request, requestDetails, message, socket);
+            std::map<std::string, std::string> replaceVar = {{"ACCESS_TOKEN",""}, {"SOCKET_PROXY", ""}, {"ACCESS_TOKEN_TTL", ""}, {"ACCESS_HEADER", ""}, {"HOST", ""},  {"VERSION", "COOLWSD_VERSION_HASH"}, {"COOLWSD_VERSION", "COOLWSD_VERSION"}, {"SERVICE_ROOT", ""}, {"UI_DEFAULTS", ""}, {"POSTMESSAGE_ORIGIN", ""}, {"PROTOCOL_DEBUG", ""}, {"HEXIFY_URL", ""}, {"BRANDING_CSS", ""}, {"BRANDING_JS", ""},  {"CSS_VARIABLES", ""}, {"DOCUMENT_SIGNING_DIV", ""} , {"DOCUMENT_SIGNING_URL", ""}, {"BROWSER_LOGGING", ""}, {"GROUP_DOWNLOAD_AS", ""},  {"OUT_OF_TIMEOUT_SECS", ""}, {"IDLE_TIMEOUT_SECS", ""}, {"ENABLE_WELCOME_MSG", ""}, {"AUTO_SHOW_WELCOME", ""}, {"UI_RTL_SETTINGS", ""},   {"USE_INTEGRATION_THEME", ""}, {"ENABLE_MACROS_EXECUTION", ""},  {"AUTO_SHOW_FEEDBACK", ""}, {"FEEDBACK_URL", "FEEDBACK_URL"}, {"WELCOME_URL", "WELCOME_URL"}, {"BUYPRODUCT_URL", ""}, {"FRAME_ANCESTORS", ""}, {"USER_INTERFACE_MODE",""}};
```

Comment:
Since we are setting the map keys and values below, I wonder what advantage is there in having default values that we later override. It seems that we only need default values for the cases that we don't have a value for. So we can simplify this greatly.

This line is excessively long. We use clang-format, which you can also use to automatically format all your code, but you can also manually break this on multiple lines, roughly making each line equal in length (about 80-100 characters long).

This helps readability and maintainability.